src/templates/best-cards/generic/animalism.html

```diff
@@ -1,115 +1,97 @@
-        Permanent intercept is well worth the cost.
-    </p>
-</div>
-<div class="column">
-    <div class="column-image">
-        <img src="{{ url_for('static', filename='img/card-images/sensethesavageway.jpg') }}">
-        <h4>Sense the Savage Way</h4>
-        <p class="center-subtitle">played in 167 decks, typically 2-6 copies</p>
-    </div>
-    <p>
-        Even if it is only available to big capacity vampires,
-        this is intercept and wake built in the same card,
-        a no-brainer if the deck has intercept options. It is even arguably better than <i>a</i>
-        {{ card("Eyes of Argus") }},
-        as the intercept can be used on an undirected action.
-    </p>
-</div>
-<div class="column">
-    <div class="column-image">
-        <img src="{{ url_for('static', filename='img/card-images/aidfrombats.jpg') }}">
-        <h4>Aid from Bats</h4>
-        <p class="center-subtitle">played in 152 decks, typically 6-14 copies</p>
-    </div>
-    <p>
-        The second pillar of <i>i</i> combat modules, it provides a maneuver, one damage, plus a welcome
-        press,
-        either as a defensive tool to avoid a second round at close range, or as an offensive tool to
-        continue combat. Played multiple times in a row, it can take an opponent down
-        pretty fast. Coupled with a {{ card("Carrion Crows") }},
-        it delivers 6 damage with only two cards.
-    </p>
-</div>
-<div class="column">
-    <div class="column-image">
-        <img src="{{ url_for('static', filename='img/card-images/armyofrats.jpg') }}">
-        <h4>Army of Rats</h4>
-        <p class="center-subtitle">played in 137 decks, typically 1-2 copies</p>
-    </div>
-    <p>
-        <i>i</i> usually makes for excellent intercept and good combat,
-        but it may lack pool damage to fence off one’s prey,
-        with {{ card("Deep Song") }}
-        bringing only a bit of pool damage.
-        This is an excellent option to improve pool damage
-        if the deck has enough intercept capabilities to protect it.
+        {{ read_winds }}.
     </p>
+	{% endtrans %}
 </div>
+
+{% call layout.card_column('Carrion Crows', decks=252, copies="5-10") %}
+{% trans trimmed %}
+An excellent combat card. It is not a strike so it cannot be dodged,
+it is harder to prevent and additional damage are still inflicted by the strike itself.
+Moreover, it is ranged damage.
+{% endtrans %}
+{% endcall %}
+
+{% call layout.card_column("Cats' Guidance", decks=222, copies="2-5") %}
+{% trans trimmed %}
+A versatile defence tool providing either intercept or an unlock effect.
+Not as suprising as a wake, but being unlocked can be better.
+{% endtrans %}
+{% endcall %}
+
+{% call layout.card_column("Canine Horde", decks=195, copies="1-3") %}
+{% trans trimmed
+	carrion_crows=card("Carrion Crows"),
+	aid_from_bats=card("Aid from Bats")
+%}
+An excellent counter against weapons, it is often included in <i>i</i> combat modules.
```

Comment:
```suggestion
An excellent counter against weapons, it is often included in {{ ani }} combat modules.
```

Same for the other occurrences of disciplines - my hope is that it will be far easier for future translators.